test/module/irohad/model/operators/model_operators_test.cpp

```diff
@@ -72,6 +73,26 @@ TEST(ModelOperatorTest, AddAssetQuantityTest) {
   ASSERT_NE(first, second);
 }
 
+// -----|SubtractAssetQuantity|-----
+
+SubtractAssetQuantity createSubtractAssetQuantity() {
+  SubtractAssetQuantity saq;
+  saq.account_id = "123";
+  iroha::Amount amount(1010, 2);
+  saq.amount = amount;
+  saq.asset_id = "123";
```

Comment:
Set `asset_id` different from `account_id`, so there's can be less typos

Reply:
ok

Reply:
Ok

PS 
as https://github.com/hyperledger/iroha/blob/d143105918b961864e52a59c800c94c1db541898/test/module/irohad/model/operators/model_operators_test.cpp#L60